src/protocol.h

```diff
@@ -291,7 +291,15 @@ enum ServiceFlags : uint64_t {
  * unless they set NODE_NETWORK_LIMITED and we are out of IBD, in which
  * case NODE_NETWORK_LIMITED suffices).
  *
- * Thus, generally, avoid calling with peerServices == NODE_NONE.
+ * Thus, generally, avoid calling with peerServices == NODE_NONE, unless
+ * state-specific flags must absolutely be avoided. When called with
+ * peerServices == NODE_NONE, the returned desirable service flags are
+ * guaranteed to not change dependant on sate - ie they are suitable for
```

Comment:
I love sate.

Reply:
I am very sate-iated...err, no, that doesnt work either...will fix.

Reply:
Fixed.